src/index.js

```diff
@@ -158,21 +158,9 @@ function jumpToPast (history, index) {
 }
 // /jumpToPast
 
-// wrapState: for backwards compatibility to 0.4
-function wrapState (state) {
-  return {
-    ...state,
-    history: state
-  }
-}
-// /wrapState
-
 // updateState
 function updateState (state, history) {
```

Comment:
I think it need, this function is simple just for  `return {...state, ...history}`. And in the `undoable` function, using the  `updateState ` to update the state after `undo`, `redo`...
```
switch (action.type) {
      case config.undoType:
        res = undo(state)
        debug('after undo', res)
        debugEnd()
        return res ? updateState(state, res) : state
```
If you want to remove it, you can replace `updateState(state, res)` with `{...state, ...res}`. And what do u think? 😀